include/onnxruntime/core/graph/graph.h

```diff
@@ -651,6 +657,9 @@ class Node {
 
   // Graph instances for subgraphs that are owned by this Node
   std::vector<std::unique_ptr<Graph>> subgraphs_;
+
+  // List of removable attributes.
+  std::set<std::string> removable_attributes_;
```

Comment:
>std::set<std::string [](http://example.com/codeflow?start=2&length=20)

Please, use IninedHashSet

Reply:
It is an InlinedVector. The HashSet is not needed anymore.